test/test_bar.py

```diff
@@ -103,7 +105,18 @@ def test_completion():
     c.reset()
     assert c.complete("/bin") != "/bin/"
     c.reset()
-    assert c.complete("~") != "~"
+
+    home_dir = os.path.expanduser("~")
```

Comment:
Hmm, this is interesting that we would change the tests depending on whether or not there is something is home. Maybe it would be better to instead create a temporary directory and test the different behaviors we expect on files that we have more control over?

Reply:
Good idea, I've update it so that both the cases of empty dir and non-empty dir can be covered.